src/components/genetic-variants/genetic-variants.tsx

```diff
@@ -297,6 +302,12 @@ export class GeneticVariants implements ComponentInterface {
   readonly EXPRESSION_RANGE: string = this.EXPRESSION_BASE + ".valueRange.select(iif($this.low.value.exists(), $this.low.value.toString(), '') + '-' + iif($this.high.value.exists(), $this.high.value.toString(), ''))";
   readonly EXPRESSION_INTEGER: string = this.EXPRESSION_BASE + ".valueInteger";
   readonly EXPRESSION_STRING: string = this.EXPRESSION_BASE + ".valueString";
+  readonly CHROMOSOME: string = "Observation.component.where(code.coding.system='http://loinc.org' and code.coding.code='48001-2').valueCodeableConcept.coding.iif($this.display.exists(), $this.display, $this.code)";
```

Comment:
please reuse EXPRESSION_STRING for this. You can replace the values as in line 390. same for the next expressions